templates/influxdata.repo.j2

```diff
@@ -1,7 +1,7 @@
 [influxdb]
-name = InfluxData Repository - {{ ansible_distribution }} $releasever
-baseurl = https://repos.influxdata.com/{{ ansible_distribution|lower }}/$releasever/$basearch/stable
+name = InfluxData Repository - Stable
+baseurl = https://repos.influxdata.com/stable/\$basearch/main
```

Comment:
The original didn't have a backslash before the `$`, but that seems like a bug in the template that you're fixing here?

Reply:
Thanks - good catch, I've removed that as this is the raw file and no need to escape it.